mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_service.go

```diff
@@ -0,0 +1,185 @@
+package apphub
+
+import (
+    "fmt"
+    "time"
+
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+    "github.com/hashicorp/terraform-provider-google/google/tpgresource"
+    transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func DataSourceApphubDiscoveredService() *schema.Resource {
+    return &schema.Resource{
+        Read: dataSourceApphubDiscoveredServiceRead,
+        Schema: map[string]*schema.Schema{
+                        "location":{
```

Comment:
Nit: This is indented more than it should be. Some of the indentation for subfields could also be made to be consistent.